CHARTER.md

```diff
@@ -0,0 +1,137 @@
+
+- Testing frameworks (e.g., unit, integration, or end-to-end testing tools).
+- APIs and tools that uses webpack in any way and are not in the webpack organization.
+- Server-side rendering directly (although it can integrate with SSR frameworks).
+- `webpack`-related frameworks and libraries that operate in their own capacity.
+
+## Section 2. Relationship with OpenJS Foundation CPC.
+
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership
+for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS
+Cross Project Council (CPC). In the case of the webpack project, it is delegated to the webpack Technical Steering
+Committee ("TSC"). OpenJS Foundation's business leadership is the Board of Directors (the "Board").
+
+This charter can only be amended with the approval of the CPC.
+
+## Section 3. Establishment of the TSC
+
+TSC members can be either _regular_ members or _voting_ members. Regular members can attend meetings and participate in
+TSC discussions, but do not vote. Voting members can do everything regular members can do, and also have the ability to
+cast votes when consensus is not reached on an issue.
+
+TSC memberships are not time-limited. There is no maximum size of the TSC. The TSC must have at least four voting
+members.
+
+The TSC may add additional voting members to the TSC through meeting consensus. The consensus requires at least five TSC members to be present to approve a new voting member.  A TSC member can be removed from the TSC by voluntary
+resignation or by consensus with at least five voting members to be present. A vote in a TSC meeting can be used to change a regular TSC member to a voting
+TSC member, or to change a voting TSC member to a regular TSC member.
+
+No more than one-fourth of the TSC voting members may be affiliated with the same employer. If a change in TSC voting
+membership or a change of employment by a TSC voting member creates a situation where more than one-fourth of the TSC
+voting membership shares an employer, then the situation must be immediately remedied by the removal of voting member
+status from one or more TSC voting members affiliated with the over-represented employer(s).
+
+The TSC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On
+Air, or through any other appropriate means selected by the TSC). The meeting shall be directed by the TSC Chairperson.
+Responsibility for directing individual meetings may be delegated by the TSC Chairperson to any other TSC voting member.
+Minutes or an appropriate recording shall be taken and made available to the community through accessible public
+postings.
+
+TSC voting members are expected to regularly participate in TSC meetings and issue triaging. Meeting notes will be released such that absent members of the TSC can provide feedback in any way.
```

Comment:
```suggestion
TSC voting members are expected to regularly participate in TSC meetings and issue triaging.
```

Seems like the second part of that paragraph is a little redundant with the minute taking noted above.